backends/redis.go

```diff
@@ -72,6 +72,7 @@ func NewRedisBackend(cfg config.Redis) *RedisBackend {
 
 	if err != nil {
 		log.Fatalf("Error creating Redis backend: %v", err)
+		panic("RedisBackend failure. This shouldn't happen.")
```

Comment:
Other backends panic after failing to be created. I think it also makes sense in the Redis case